mocks/Client.go

```diff
@@ -286,6 +286,35 @@ func (_m *Client) SignalWorkflow(ctx context.Context, workflowID string, runID s
 	return r0
 }
 
+// SignalWithStartWorkflow provides a mock function with given fields: ctx, options, workflow, args
```

Comment:
SignalWithStartWorkflow provides a mock function with given fields: ctx, options, workflow, args ->
SignalWithStartWorkflow provides a mock function with the following fields: ctx, options, workflow, args.